src/screens/UserPortal/Volunteer/Groups/GroupModal.tsx

```
@@ -187,225 +186,239 @@ const GroupModal: React.FC<InterfaceGroupModal> = ({
+            onClick={hide}
+            className={styles.modalCloseBtn}
+            data-testid="modalCloseBtn"
+          >
+            <i className="fa fa-times"></i>
+          </Button>
+        </div>
+      }
+    >
+      <div
+        className={`btn-group ${styles.toggleGroup} mt-0 px-3 mb-4 w-100`}
+        role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)